styles.css

```diff
@@ -372,6 +372,173 @@ a {
   color: #fff;
 }
 
+/* Modal */
+
+.modal-section {
+  display: none;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  width: 100%;
+  z-index: 1;
+  left: 50%;
+  transform: translate(-50%, 0);
+  max-height: calc(150vh - 210px);
+  overflow-y: auto;
+  box-sizing: content-box;
+}
+
+.modal {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  mix-blend-mode: multiply;
+  padding: 1.25rem 1rem;
+  width: 80%;
+  background: #c1c7d0;
+}
+
+.card-work {
+  flex-direction: column;
+  justify-content: center;
+  align-self: center;
+  padding: 1rem;
+  gap: 0.7rem;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 1rem;
+}
+
+.modal-header {
+  border-radius: 0.5rem;
+  width: 100%;
+  margin: 0;
+}
+
+.project-image {
+  width: 100%;
+  height: 40vh;
+  background-size: contain;
+}
```

Comment:
- [ ]  Your portfolio is looking great! However, the modal window image is getting distorted when the viewport size changes because `height` is depending on the window's vertical size and `width` is depending on the window's horizontal size, therefore they are working separately:


|1|2|
|---|---|
|![Screen Shot 2022-09-07 at 1 43 11 PM](https://user-images.githubusercontent.com/83717285/188954365-63cc0d12-8b9b-4d77-8d64-583947e722fc.png)|![Screen Shot 2022-09-07 at 1 43 50 PM](https://user-images.githubusercontent.com/83717285/188954392-482dd275-20f9-458d-bfff-489f2eedb28d.png)|

You could try with `max-width: 100%` and/or `height: 100%` and check what happens.

Reply:
Well resolved 👍🏽